src/translator/byte_array_util.cpp

```diff
@@ -17,29 +18,41 @@ void * getBinaryFile(std::string path) {
         std::exit(1);
     }
     void *result;
-    int fail = posix_memalign(&result, 256, length);
-    if (fail) {
-        std::cerr << "Failed to allocate aligned memory." << std::endl;
-        std::exit(1);
+    if (isModelFile) {
```

Comment:
Seeing as to alignment to 256 is checked during model construction, can we shift this responsibility to the model constructor (from blob) and make this function cleaner? Maybe move as an std::string (fpath) based constructor to `MemoryGift`?